x-pack/plugins/ml/public/application/components/data_grid/data_grid.tsx

```diff
@@ -188,6 +188,43 @@ export const DataGrid: FC<Props> = memo(
       );
     }
 
+    let errorCallout;
+
+    if (status === INDEX_STATUS.ERROR) {
+      // if it's a searchBar syntax error leave the table visible so they can try again
+      if (!errorMessage.includes('failed to create query')) {
+        errorCallout = (
+          <EuiCallOut
+            title={i18n.translate(
+              'xpack.ml.dataframe.analytics.regressionExploration.querySyntaxError',
```

Comment:
Is this error specific to `regressionExploration`?

Reply:
It is not - good catch. Updated to just `exploration` in 08b88797de85c4faabcc17f9b7d2488ab4b375ae 👍